issue-pr-templates/bin/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Questions:
+  https://groups.google.com/forum/#!forum/loopbackjs
+  https://gitter.im/strongloop/loopback
+Immediate support:
+  https://strongloop.com/api-connect-faqs/
+  https://strongloop.com/node-js/subscription-plans/
+-->
+
+# Description/Steps to reproduce
+
+<!--
+If feature: A description of the feature
+If bug: Steps to reproduce
+-->
```

Comment:
I did add a header to link sandbox repo couple of lines [down](https://github.com/strongloop-internal/loopback-scripts/pull/6/files#diff-a8300ae96c4fcdba614fc3f7a8970546R17)